quickstart-linux.yaml

```diff
@@ -226,6 +226,26 @@ data:
               - "extnet:name=internet"
             {{- end}}
           {{- end}}
+          {{- if .Values.protection_incoming_ssh }}
+          - name: "Allow incoming SSH access"
+            description: "This policy has been created from the linux workflow"
+            action: "Allow"
+            logsEnabled: true
+            {{- if .Values.create_new_namespace }}
+            namespace: "{{ .Values.namespace_name }}"
+            subject:
+            - - "$name=Internet (any TCP)"
```

Comment:
Use the associated tags instead of the name. Name is not secure as if you have another object with the same name, it will match

Reply:
updated